### PR TITLE
Created the footer and newsletter

### PR DESCRIPTION
**Pull request by Gabriel0508**
**Developed by:**

-created the footer
-created the newsletter

image

### CODE REVIEW

File: src/app/pages/newsletter/newsletter/newsletter.component.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        >
+          Email required
+        </span>
+
+        <span
+          *ngIf="(newsletterForm.controls['email'].touched || newsletterForm.controls['email'].dirty) && newsletterForm.controls['email'].errors?.['email']"
+          class="err-message"
+        >
+          Enter a valid email address
+        </span>
```
Comment (alexPopNicolae) [inline src/app/pages/newsletter/newsletter/newsletter.component.html]: this should be abstracted in a single message as we did on the register form.

File: src/app/pages/newsletter/newsletter/newsletter.component.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    <form [formGroup]="newsletterForm" (ngSubmit)="onSubmit()" class="newsletter">
+      <div class="email">
+        <input
+          class="newsletter-input"
+          type="text"
+          formControlName="email"
+          placeholder="example@gmail.com"
+        />
+        <span
+          *ngIf="(newsletterForm.controls['email'].touched || newsletterForm.controls['email'].dirty) && newsletterForm.controls['email'].errors?.['required']"
```
Comment (alexPopNicolae) [inline src/app/pages/newsletter/newsletter/newsletter.component.html]: this should be abstracted in a single message as we did on the register form.

File: src/app/shared/footer/footer.component.scss [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  .nav-links {
+    list-style: none;
+    display: flex;
+    justify-content: space-between;
+    .create,
+    .follow,
+    .support {
+      .title {
+        position: relative;
+        font-family: "Open Sans";
```
Comment (alexPopNicolae) [inline src/app/shared/footer/footer.component.scss]: asta ar trebui scris asa

font-family: Open-sans;

de asta nu iti arata ca in design

"Open Sans" nu e la fel ca Open-sans - asta pentru ca noi l-am definit intr-un font face

File: src/app/shared/footer/footer.component.scss [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+      .link {
+        display: flex;
+        align-items: center;
+        margin: 10px 0;
+        .icon {
+          width: 20%;
+        }
+        .link-item {
+          text-decoration: none;
+          font-family: "Open Sans";
```
Comment (alexPopNicolae) [inline src/app/shared/footer/footer.component.scss]: acelasi lucru ca mai sus legat de fonturi - foloseste Open-sans

File: src/app/pages/newsletter/newsletter/newsletter.component.scss [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+        width: 41px;
+        height: 3px;
+        background: #1d741b;
+        position: absolute;
+        bottom: 0;
+        border-radius: 2px;
+      }
+    }
+    .description {
+      font-family: "Open Sans";
```
Comment (alexPopNicolae) [inline src/app/pages/newsletter/newsletter/newsletter.component.scss]: foloseste Open-sans

File: src/app/pages/newsletter/newsletter/newsletter.component.scss [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+      width: 100%;
+      .newsletter-input {
+        height: 94px;
+        padding: 0 20px;
+        border: 1px solid #e3e3e3;
+        outline: none;
+        border-radius: 15px;
+        background: #e3e3e3;
+        font-family: "Open Sans";
+        font-style: normal;
```
Comment (alexPopNicolae) [inline src/app/pages/newsletter/newsletter/newsletter.component.scss]: si aici tot la fel legat de font

File: src/app/pages/newsletter/newsletter/newsletter.component.scss [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+      .btn-primary {
+        width: 311px;
+        height: 94px;
+        border: 1px solid #ca3c0a;
+        background: #ca3c0a;
+        color: #ffffff;
+        padding: 10px;
+        border-radius: 15px;
+        cursor: pointer;
+        font-family: "Open Sans";
```
Comment (alexPopNicolae) [inline src/app/pages/newsletter/newsletter/newsletter.component.scss]: acelasi lucru legat de font

File: src/app/pages/landing/landing.module.ts [hunk 8]
```diff
@@ -2,16 +2,16 @@ import { NgModule } from '@angular/core';
...
 
 @NgModule({
   declarations: [
     LandingComponent
   ],
   imports: [
     CommonModule,
-    SharedModule
+    SharedModule,
+    NewsletterModule
```
Comment (alexPopNicolae) [inline src/app/pages/landing/landing.module.ts]: newsletterul nu e o componenta asa de mare ... si ar trebui pusa in shared module...
nu are nevoie de propriul modul - pune componenta impreuna cu restul in shared module

File: src/app/pages/newsletter/newsletter/newsletter.component.scss [hunk 9]
```diff
@@ -0,0 +1,92 @@
+.newsletter-container {
+  width: 100%;
+  height: 50vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  .text {
+    .title {
+      position: relative;
+      font-family: "Open Sans";
```
Comment (alexPopNicolae) [inline src/app/pages/newsletter/newsletter/newsletter.component.scss]: schimba fontul

**Review by alexPopNicolae — CHANGES_REQUESTED** [review]

File: src/app/pages/newsletter/newsletter/newsletter.component.html (hunk 1 above)
Comment (Gabriel0508) [reply src/app/pages/newsletter/newsletter/newsletter.component.html]: done

File: src/app/pages/newsletter/newsletter/newsletter.component.html (hunk 2 above)
Comment (Gabriel0508) [reply src/app/pages/newsletter/newsletter/newsletter.component.html]: done

File: src/app/shared/footer/footer.component.scss (hunk 3 above)
Comment (Gabriel0508) [reply src/app/shared/footer/footer.component.scss]: done

File: src/app/pages/newsletter/newsletter/newsletter.component.scss (hunk 5 above)
Comment (Gabriel0508) [reply src/app/pages/newsletter/newsletter/newsletter.component.scss]: done

File: src/app/pages/newsletter/newsletter/newsletter.component.scss (hunk 7 above)
Comment (Gabriel0508) [reply src/app/pages/newsletter/newsletter/newsletter.component.scss]: done

File: src/app/pages/newsletter/newsletter/newsletter.component.scss (hunk 9 above)
Comment (Gabriel0508) [reply src/app/pages/newsletter/newsletter/newsletter.component.scss]: done

File: src/app/pages/landing/landing.module.ts (hunk 8 above)
Comment (Gabriel0508) [reply src/app/pages/landing/landing.module.ts]: done

File: src/app/shared/footer/footer.component.scss (hunk 4 above)
Comment (Gabriel0508) [reply src/app/shared/footer/footer.component.scss]: done

File: src/app/pages/newsletter/newsletter/newsletter.component.scss (hunk 6 above)
Comment (Gabriel0508) [reply src/app/pages/newsletter/newsletter/newsletter.component.scss]: done

**Review by alexPopNicolae — APPROVED** [review]

**Pull request merged**